.github/workflows/app-etterlatte-beregning-kafka.yaml

```diff
@@ -55,3 +55,6 @@ jobs:
     with:
       image: ${{ needs.build.outputs.image }}
     secrets: inherit
+    permissions:
```

Comment:
Skal det gå ann å sette disse permissionsene på joben og ikke på hver build og deploY?

Reply:
Oh, liker tanken. Usikker, men lov å håpe, så skal teste det.

Reply:
Nice, det skal gå ref https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#example-setting-the-github_token-permissions-for-an-entire-workflow

Reply:
Ser sånn ut: https://github.com/navikt/pensjon-etterlatte-saksbehandling/actions/runs/7260166168/job/19778776271